### DISCUSSION TITLE
Criando Apps Web com Angular e UiKit - Aula 16

### DISCUSSION THREAD

**Discussion by Pakato14** [balta-io/forum#255] [Tira Dúvidas] [answered]
Olá, estou fazendo o projeto do petshop e o ngClass está indicando um erro  ao digitar username

`[ngClass]="{'uk-form-danger': (!form.controls.username.valid && !form.controls.username.pristine) }"`

como posso resolver esse problema?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @Pakato14, como vai?
Qual erro está sendo indicado?

**Comment by Pakato14** [comment]
Aparece essa mensagem: 

`Property 'username' comes from an index signature, so it must be accessed with ['username'].`

**Reply by andrebaltieri** [reply]
@Pakato14, como vai?
Tenta isto:
```
[ngClass]="{'uk-form-danger': (!form.controls['username'].valid && !form.controls.username.pristine) }"
```
